client/src/components/Login.js

```diff
@@ -57,8 +57,24 @@ signupLink:{
 }));
 function Login (){
     const classes = useStyles();
-    const [userEmail,setEmail]=useState('')
-    const [userPass, setPass]=useState('')
+    const [email,setEmail]=useState('')
+    const [password, setPass]=useState('')
+    const handleClick = () => {
+        fetch("/auth/login", {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({
+                'email': email,
+                'password': password,
+            })})
+            .then(response => response.json())
+            .then(console.log('Success')) // Logs success even when unsuccessful, not certain how to fix it.
```

Comment:
What we want to do here is look at the http status code that's in the response - if it's 200, we're good to go. if it's not, then we'd look at the message of the response to get the right error message to display